examples/lead-agent/backend/app.py

```
@@ -0,0 +1,147 @@
+        # Run the lead scoring flow
+        result = client.run(
+            top_n=top_n,
+            job_description=job_description,
+            additional_instructions=additional_instructions,
+            generate_emails=generate_emails,
+            candidates=candidates
+        )
+        
+        return jsonify(result), 200
+        
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)